frontend/src/components/ingress/Details.tsx

```diff
@@ -42,6 +43,110 @@ export default function IngressDetails() {
     );
   }
 
+  /** this function is used to determine the protocol used in the formatURL for LinkedStringFormat */
+  function getProtocol(item: any, url: any) {
```

Comment:
This function is called `getProtocol` which suggests it returns the protocol, but it returns whether https is used.